README.md

```diff
@@ -34,139 +34,275 @@ For the moment, only one client can be instantiated globally per process.
 Future releases will allow multiple client instantiations.
 
 ```js
-import { createClient } from 'tigerbeetle-node'
+const {
+  createClient,
+  CreateAccountError,
+  CreateAccountFlags,
+  CreateTransferFlags,
+  CreateTransferError,
+} = require('tigerbeetle-node');
 
 const client = createClient({
   cluster_id: 0,
   replica_addresses: ['3001', '3002', '3003']
-})
+});
 ```
 
-One of the ways TigerBeetle achieves its performance is through batching. 
-This is reflected in the below function interfaces where each one takes in an array of events.
+One of the ways TigerBeetle achieves its performance is through
+batching. This is reflected in the below function interfaces where
+each one takes in an array of items.
 
-### Account Creation
+### Sidenote: `BigInt`
+
+TigerBeetle uses 64-bit integers for many fields while JavaScript's
+builtin `Number` maximum value is 2^53-1. The `n` suffix in JavaScript
+means the value is a BigInt. This is useful for literal numbers. If
+you already have a `Number` variable though, you can call the `BigInt`
+constructor to get a `BigInt` from it. For example, `1n` is the same
+as `BigInt(1)`.
+
```

Comment:
Maybe add a warning that actual UUIDs (e.g. from Postgres) shouldn't be converted to numbers since they could lose precision.

Reply:
I'm not saying anything in this paragraph about converting *to Numbers* though